internal/control_plane/command/root.go

```diff
@@ -0,0 +1,63 @@
+package command
+
+import (
+	"octavius/internal/control_plane/command/start"
+	"octavius/internal/logger"
+	"os"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+var rootCmd = &cobra.Command{
+	Use:   "ControlPlane",
+	Short: "Control plane of octavius",
+	Long:  `Control plane of octavius takes request from cli`,
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		logger.Error("error in execute", err)
```

Comment:
Do we have something like `logger.Fatal` ? as it will be easier instead of log and then `os.Exit(1)`